LoopKitUI/View Controllers/AuthenticationViewController.swift

```diff
@@ -39,7 +39,8 @@ public final class AuthenticationViewController<T: ServiceAuthenticationUI>: UIT
                         }) 
 
                         if let error = error {
-                            self.presentAlertController(with: error)
+                            let alert = UIAlertController(with: error)
```

Comment:
I think this could just be considered a style preference.  In this case I might agree that a single line is at least as readable, if not more.  Just a tradeoff of density vs line count.

Reply:
:+1: